sphinx_gallery/tests/test_full.py

```diff
@@ -49,6 +60,5 @@ def test_embed_links():
     assert 'scipy.signal.firwin.html' in lines
     assert '#module-numpy' in lines
     assert 'numpy.arange.html' in lines
-    # The matplotlib doc download is a bit unsafe, so skip for now:
-    # assert '#module-matplotlib.pyplot' in lines
-    # assert 'pyplot_api.html' in lines
+    assert '#module-matplotlib.pyplot' in lines
```

Comment:
I had these commented out for a reason :)

https://travis-ci.org/sphinx-gallery/sphinx-gallery/jobs/292615909#L1129

Reply:
Yeah I did not understand why and it was passing locally so I figured I would give it a go ... oh well, maybe I'll comment them out again ... maybe it depends on the sphinx version ?

Reply:
Yeah probably sphinx. I guess you could check whether version 1.5+ is used and only check if that's the case, as I think that Ubuntu build uses 1.4.9

Reply:
For the record I could not reproduce this locally with sphinx 1.4.9. I thought maybe the matplotlib version has something to do with it (and the possible mismatch with the matplotlib doc fetched from the internet but I did not manage to reproduce either).

Reply:
Yeah at one point I tried to reproduce, as well. To really find the problem I figured I'd need to get a Trusty system going with system Python packages to mirror those of Travis, which was too much for me to follow through on.